cartoframes/data/services/bq_datasets.py

```diff
@@ -225,3 +288,13 @@ def import_dataset(self):
 
     def upload_dataframe(self, dataframe):
         return self._client.upload_dataframe(dataframe, self._name_id)
+
+    def enrichment(self, geom_type='points', variables=None, output_name=None):
```

Comment:
It is very useful. When we will add the `filters` and `aggregation` parameters, it will be easy to do it (including validation or other tasks)